src/C-interface/dense/bml_add_dense_typed.c

```diff
@@ -46,9 +47,15 @@ void TYPED_FUNC(
     int startIndex = B->domain->localDispl[myRank];
     int inc = 1;
 
+#ifdef NOBLAS
```

Comment:
Why exit? Is that so the code compiles if there is no BLAS?

Reply:
If exit is not there the code will give an error at runtime. In this case, if NOBLAS, the
code will give a message and then exit. 
What should I put instead?